app/src/main/java/mozilla/lockbox/presenter/ItemMutationPresenter.kt

```diff
@@ -146,6 +150,6 @@ abstract class ItemMutationPresenter(
     }
 
     abstract fun dismissChangesAction(hasChanges: Boolean): Action
-    abstract fun saveChangesAction(hasChanges: Boolean): ItemDetailAction
+    abstract fun saveChangesAction(hasChanges: Boolean): List<Action?>
```

Comment:
This looks as though it could be non-optional.

Also: consider changing the name to `saveChangeActions`, since this is a list now.